addons/website/static/src/components/dialog/edit_menu.js

```diff
@@ -184,14 +184,21 @@ export class EditMenuDialog extends Component {
     }
 
     addMenu(isMegaMenu) {
+        const currentUrl = window.location.href;
+        const baseUrl = window.location.origin + '/event/';
+        let eventUrl = '';
+        if (currentUrl.startsWith(baseUrl)) {
+            const eventName = currentUrl.slice(baseUrl.length).split('/')[0];
+            eventUrl = `/event/${eventName}/page`;
+        }
```

Comment:
Perhaps, we should customize `MenuDialog` to fabricate url for any module concerning website_* instead of 
'/event/' path. so it would make it generic.

Reply:
this one is quite tricky.. as we have different ways to open 'Edit Menu' (From 'Site' and From website editor) which has different mechanism, so I'm overwritting in `website_event` module.

('Edit Event Menu' is a different thing from how we edit with website editor as 'Edit Event Menu' check for child ids (for submenus) while if we edit a menu from website editor it just check if its an menu then you can edit it doesn't matter if its an event page or not. `custom_menu_edit_menu` this was used get 'Edit Event Menu')


Let me know if you have any better way do implement this one, Open for discussion :smile: